From phil@ipom.com Fri Sep 30 00:23:15 2005
Message-ID: <433CE72F.6060202@ipom.com>
Date: Fri, 30 Sep 2005 00:20:15 -0700
From: Phil Dibowitz <phil@ipom.com>
To: Greg KH <greg@kroah.com>, Stefan Werner <dustbln@gmx.de>
Subject: [PATCH pd33]: Add unusual_dev for iBeat

This patch adds the US_FL_IGNORE_RESIDUE flag for the TrekStor i.Beat
Joy 2.0. Original version of this patch was sent by Stefan Werner
<dustbln@gmx.de> with test/rediff/etc. by me.

Signed-off-by: Phil Dibowitz <phil@ipom.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

 drivers/usb/storage/unusual_devs.h |    7 +++++++
 1 file changed, 7 insertions(+)

--- gregkh-2.6.orig/drivers/usb/storage/unusual_devs.h	2005-10-03 15:04:33.000000000 -0700
+++ gregkh-2.6/drivers/usb/storage/unusual_devs.h	2005-10-03 15:04:48.000000000 -0700
@@ -106,6 +106,13 @@
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_FIX_INQUIRY ),
 
+/* Reported by Stefan Werner <dustbln@gmx.de> */
+UNUSUAL_DEV(  0x0419, 0xaaf6, 0x0100, 0x0100,
+		"TrekStor",
+		"i.Beat Joy 2.0",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_IGNORE_RESIDUE ),
+
 /* Reported by Olaf Hering <olh@suse.de> from novell bug #105878 */
 UNUSUAL_DEV(  0x0424, 0x0fdc, 0x0210, 0x0210,
 		"SMSC",
